pyspatialopt/models/binary_mclp_distance_matrix.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: UTF-8 -*-
+import logging
+import sys
+import arcpy
+import pulp
+import csv
+import sys
+import os
+from pyspatialopt.models import utilities
+from pyspatialopt.models import covering
+
```

Comment:
Add an extra new line to follow [Pep8](https://www.python.org/dev/peps/pep-0008/) styling guidelines